hook-consumer/Cargo.toml

```diff
@@ -6,3 +6,16 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+async-std = { version = "1.12" }
+chrono = { version = "0.4" }
+envconfig = { version = "0.10" }
+futures = "0.3"
+hook-common = { path = "../hook-common" }
+http = { version = "0.2" }
+reqwest = { version = "0.11" }
+serde = { version = "1.0" }
+serde_derive = { version = "1.0" }
+sqlx = { version = "0.7", features = [ "runtime-tokio", "tls-native-tls", "postgres", "uuid", "json", "chrono" ] }
```

Comment:
I agree. I was focusing on the consumer and forgot we were in a workspace. Should be quick to add, hold on!

Reply:
Actually, since this is being worked on #2, I would leave it for later to avoid conflicts. We can merge that PR and then I'll either rebase this one if it's not merged yet, or open a new one to use common dependencies.